data/metadata.sssom.yml

```diff
@@ -1,5 +1,10 @@
-mapping_set_id: http://example.org
-license: http://w3id.org/sssom/license/unspecified
+mapping_set_id: http://purl.obolibrary.org/obo/mondo/mondo-ingest/mapping/omim.sssom.tsv
+mapping_set_description: >
+  The file `omim.sssom.tsv` is generated using the Python package `sssom`, by running the command `make sssom` from a
+  cloned copy of the OMIM ingest (https://github.com/monarch-initiative/omim). For more information on data sources,
+  assumptions, and computations, refer to `README.md` or the comments at the top of `omim2obo/main.py` in the OMIM
```

Comment:
Can we get 2 sentences that summarises the processing in omim2obo/main.py? The key bits are: which OMIM datafiles are the source of the mappings, and which relationships do we assume (skos:exactMatch).

Reply:
Will do!